README.md

```diff
@@ -18,37 +18,105 @@ or
     pip install klaviyo
 
 
-## Tracking events in Python
+## API Examples
 
-You can then easily use Klaviyo to track events or identify people:
+After installing the klaviyo package you can initiate it using your public token which is for track events or identifying profiles and/or your private api key to utilize the metrics and list apis.
 
     import klaviyo
 
-    client = klaviyo.Klaviyo('YOUR_KLAVIYO_API_TOKEN')
-    
+    client = klaviyo.Klaviyo(public_token=PUBLIC_TOKEN, private_token=PRIVATE_TOKEN)
+
+You can then easily use Klaviyo to track events or identify people.  Note, track and identify requests take your public token.
+
     # Track an event...
     client.track('Filled out profile', email='someone@example.com', properties={
         'Added social accounts' : False,
     })
+    
+    # you can also add profile properties
+    client.track(
+      'Filled out profile', 
+      email='someone@example.com', 
+      properties={
+        'Added social accounts' : False,
+      }, 
+      customer_properties={
+        '$first_name': 'Thomas',
+        '$last_name': 'Jefferson'
+      }
+    )
 
     # ...or just add a property to someone
-    client.identify(email='someone@example.com', properties={
+    client.identify(email='thomas.jefferson@example.com', properties={
+        '$first_name': 'Thomas',
+        '$last_name': 'Jefferson',
         'Plan' : 'Premium',
     })
 
```

Comment:
Lets not use example.com email addresses as people will be confused when it does not work.